src/lib/rest/rest.cpp

```diff
@@ -240,7 +240,7 @@ static int uriArgumentGet(void* cbDataP, MHD_ValueKind kind, const char* ckey, c
   {
     containsForbiddenChars = forbiddenChars(val, ";");
   }
-  else if ((key != "q") && (key != "idPattern"))
+  else if ((key != "q") && (key != "mq") && (key != "idPattern"))
```

Comment:
Bah, it was 'q' before, I just added mq ...
Time to fix them, and "idPattern" also.